recipe/meta.yaml

```diff
@@ -1,9 +1,3 @@
-{% if cuda_compiler_version in (None, "None", True, False) %}
-{% set cuda_major = 0 %}
-{% else %}
-{% set cuda_major = environ.get("cuda_compiler_version", "11.8").split(".")[0]|int %}
-{% endif %}
```

Comment:
This is no longer needed as we now use `pin_compatible` to constrain `cudatoolkit` instead of rolling our own solution